core/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
     compileSdkVersion 31
 
     defaultConfig {
-        minSdkVersion 21
+        minSdkVersion 23
```

Comment:
shouldn't this be 24?
cause web3j min is 24

Reply:
core-sdk  internally has check for Android API level like in Web3Auth.kt like :
 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
            initiateKeyStoreManager()
   }

So keyStore and web3J library will work only for and above Android API level 24. Anyways we can update sdk to support min API level 24.